plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerTypeMapping.java

```diff
@@ -836,6 +837,61 @@ private void testSqlServerDatetimeOffset(ZoneId sessionZone)
                 .execute(getQueryRunner(), session, sqlServerCreateAndInsert("test_sqlserver_datetimeoffset"));
     }
 
+    @Test
+    public void testSqlServerDatetimeOffsetOldDates()
```

Comment:
nit: Instead of `old` we could use `historicalDates` as `old` is pretty relative